src/lib/falcor/routes/staff/by-slug.js

```diff
@@ -14,19 +15,21 @@ export default [
     get: pathSet =>
       new Promise(resolve => {
         const requestedFields = pathSet[3];
-        db.staffQuery(pathSet.slugs, requestedFields).then(data => {
-          // always returns slug in the object no matter what.
-          const results = [];
-          data.forEach(staff => {
-            requestedFields.forEach(field => {
-              results.push({
-                path: ['staff', 'bySlug', staff.slug, field],
-                value: staff[field],
+        staffQuery(db.database, 'slug', pathSet.slugs, requestedFields).then(
```

Comment:
Use async/await here.

Reply:
That's a rabbit hole to go down where I would end up refactoring 2000 lines of code, trust me I feel like it, but all I did here was change the way a function was imported